include/zephyr/bluetooth/services/ans.h

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2025 Sean Kyer
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_SERVICES_ANS_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_SERVICES_ANS_H_
+
+/**
+ * @brief Alert Notification Service (ANS)
+ * @defgroup bt_ans Alert Notification Service (ANS)
+ *
+ * @since 4.3
+ * @version 0.1.0
+ *
+ * @ingroup bluetooth
+ * @{
+ */
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Command not supported error code
+ */
+#define BT_ANS_ERR_CMD_NOT_SUP 0xa0
+
+/**
+ * @brief ANS max text string size in octets
+ *
+ * This is the max string size in octets to be saved in a New Alert. Text longer than the max is
+ * truncated.
+ *
+ * section 3.165 of
+ * https://btprodspecificationrefs.blob.core.windows.net/gatt-specification-supplement/GATT_Specification_Supplement.pdf
+ *
+ */
+#define BT_ANS_MAX_TEXT_STR_SIZE 18
+
+/**
+ * @brief ANS Category ID Enum
+ *
+ * Enumeration for whether the category is supported.
+ */
+enum bt_ans_cat {
+	BT_ANS_CAT_SIMPLE_ALERT,    /**< Simple alerts (general notifications). */
+	BT_ANS_CAT_EMAIL,           /**< Email messages. */
+	BT_ANS_CAT_NEWS,            /**< News updates. */
+	BT_ANS_CAT_CALL,            /**< Incoming call alerts. */
+	BT_ANS_CAT_MISSED_CALL,     /**< Missed call alerts. */
+	BT_ANS_CAT_SMS_MMS,         /**< SMS/MMS text messages. */
+	BT_ANS_CAT_VOICE_MAIL,      /**< Voicemail notifications. */
+	BT_ANS_CAT_SCHEDULE,        /**< Calendar or schedule alerts. */
+	BT_ANS_CAT_HIGH_PRI_ALERT,  /**< High-priority alerts. */
+	BT_ANS_CAT_INSTANT_MESSAGE, /**< Instant messaging alerts. */
+
+	BT_ANS_CAT_NUM, /**< Marker for the number of categories. */
```

Comment:
We should consider omitting this particular value from the documentation

```suggestion
	/** @cond INTERNAL_HIDDEN */
	BT_ANS_CAT_NUM, /**< Marker for the number of categories. */
	/** @endcond */
```